src/Restaurants.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+function Restaurants({ restaurants }) {
+  if (!restaurants) {
+    return <></>;
+  }
+  return (
+    <ul>
+      {restaurants.map((restaurant) => <li key={restaurant.id}>{restaurant.name}</li>)}
+    </ul>
```

Comment:
```suggestion
    <ul>
      {restaurants.map((restaurant) => (
        <li key={restaurant.id}>
          {restaurant.name}
        </li>
      ))}
    </ul>
```

제 생각에는 한줄로 긴 경우에 줄바꿈을 해주면 가독성에 더 좋을 것 같은데 어떤게 더 나은가요?